code/modules/mob/living/simple_animal/hostile/mining_mobs/mining_mobs.dm

```diff
@@ -42,12 +42,12 @@
 		return
 	icon_state = icon_living
 
-/mob/living/simple_animal/hostile/asteroid/bullet_act(obj/projectile/P)//Reduces damage from most projectiles to curb off-screen kills
+/mob/living/simple_animal/hostile/asteroid/bullet_act(obj/projectile/shot)//Reduces damage from most projectiles to curb off-screen kills
 	if(!stat)
 		Aggro()
-	if(P.damage < 30 && P.damage_type != BRUTE)
-		P.damage = (P.damage / 3)
-		visible_message(span_danger("[P] has a reduced effect on [src]!"))
+	if(shot.damage < 30 && shot.damage_type != BRUTE || !shot.penetrate_mining_mobs)
```

Comment:
Include this change in your changelog.

Reply:
If this isn't a bug then I can go ahead and close both the issue and this PR.

Reply:
don't see why it should have an effect on mining mobs.

Reply:
Plasma cutters in their current form deal `BRUTE` damage which means they don't lose 1/3 of their damage when attacking mining mobs.  Since I switched the damage type to `BURN` so we can ignite plasma, I added a `penetrate_mining_mobs` var so that their damage vs mining mobs is not affected.

I think the crux of the issue is that "**Burn projectiles don't ignite plasma**" which falls into consistency/oversight and I can make a bug report (that is more accurate) to go with this if needed.  Otherwise I can hardcode `if(istype(shot, "/obj/projectile/plasma")` so that no other `BURN` projectile will ignite plasma but this is really bad code practice.

When I pinged maintainers on discord, I was adamant about not getting penalized for trying to fix year old bugs that nobody else wants to fix.

---

If the issue _isn't_ a bug, I'm happy to have it closed and further decrease the issue tracker.
If my code implementation affects other things _outside_ the bug, I'm happy to change my code so it doesn't affect other things.
If you need me to document the _scope_ of the issue with more accurate bug reports before fixing the issue, I can do that.

But I'm not going to fix things that are determined to be broken and then get punished for it.  Having these discussions again and again with maintainers is burning me out honestly and is making me regret trying to help... just my thoughts.  It's also why I tried to have an open discussion on discord about this, but i feel it's done in bad faith.

Reply:
I think you should just not change this part of the code at all. If mobs take reduced damage because of plasma cutters, then that's just a consequence of fixing a bug. It's not like you have to maintain the behaviour 1:1 and it's not like plasma cutters are a weapon to be used offensively anyways, so it's not a big deal for it to be affected this way.

Reply:
If you want to properly fix this, it shouldn't be tied to damage types.

Reply:
> I think you should just not change this part of the code at all. If mobs take reduced damage because of plasma cutters, then that's just a consequence of fixing a bug. It's not like you have to maintain the behaviour 1:1 and it's not like plasma cutters are a weapon to be used offensively anyways, so it's not a big deal for it to be affected this way.

That's fair.  I switched the damage type to `BURN` and added it as a balance changelog.  I admit I was in a pissy mood the other day, so you have my apologies.